api/backend/models.py

```diff
@@ -24,7 +30,9 @@ class User(db.Model):
     image = db.Column(db.Text, nullable=True)
     meta_id = db.Column(db.Integer, db.ForeignKey('meta.id'))
     meta = db.relationship("Meta", back_populates="user")
-
+    tags = db.relationship("Tag", secondary="user_tag_rel",               
```

Comment:
We should use back_populates instead of backref, since it is easier to tell which models are related to each other